apps/typegpu-docs/src/content/examples/simulation/fluid-double-buffering/index.ts

```diff
@@ -431,7 +414,7 @@ const mainCompute = tgpu['~unstable']
     const y = d.i32(input.gid.y);
     const index = coordsToIndex(x, y);
 
-    setupRandomSeed(d.vec2f(d.f32(index), timeUniform.value));
+    BPETER.seed(d.vec2f(d.f32(index), timeUniform.value));
```

Comment:
since we don't assign `BPETER.sample` to the `randomGeneratorSlot` in this example, we shouldn't use `BPETER.seed`, but rather `DefaultGenerator.seed`. I would also consider changing the slot to contain both sample and seed functions, so that user functions can use the correct seed function, taking it from `randomGeneratorSlot`